src/morris_sensitivity.jl

```diff
@@ -0,0 +1,28 @@
+function generate_design_matrix(prob::DEProblem,param_bounds,p)
+    init_param_vector = prob.p
+    B = Array{Array{Float64}}(length(prob.p)+1)
+    r = Float64[]
+    for i in 1:length(param_bounds)
+        dels = [j/p for j in param_bounds[i][1]:param_bounds[i][2]]
+        del_i = rand(dels)
+        push!(r,init_param_vector[i]+del_i)
+        # println(r)
+    end
+
```

Comment:
isn't r a sufficient representation of B?

Reply:
For the same `r` we can have different trajectories, so I don't think so?